src/js/settings.js

```diff
@@ -0,0 +1,61 @@
+import { setting } from './translation.js'
+
+const settings = document.querySelector('.settings');
+const settingsBtn = document.querySelector('.icon_settings');
+const elementsList = document.querySelectorAll('.block input');
+const elementsListText = document.querySelectorAll('.block label');
+const textCahngeLang = document.querySelector('.title__language');
+const textChangePhoto = document.querySelector('.title__photo');
+const ruLang = document.querySelector('option[value="ru"]');
+const enLang = document.querySelector('option[value="en"]');
```

Comment:
variable naming can be improved here. it looks like you didn't pay any attention to how to name these things, they look very inconsistent even though they are literally go one after another. what you can do is for example add element name to variable everywhere like you did with `settingsBtn`. I would have no idea when I read your code that for example `ruLang` refers to an actual option element and not to `ruLang` object or whatever else, so you can call it e.g. `ruLangOption` instead